docs/kubernetes.md

```diff
@@ -32,6 +32,34 @@ In the image above, you can see the following parts:
 
 All VMs are in the same private VNET and are fully accessible to each other.
 
+
+## Optional: Enable network policy enforcement using Calico
+
+Using the default configuration, Kubernetes allows communication between all
+Pods within a cluster. To ensure that Pods can only be accessed by authorized
+Pods, a policy enforcement is needed. To enable policy enforcement using Calico
+`azuredeploy.parameters.json` needs to be modified like that:
+
+```json
+"networkPolicy": {
+  "value": "calico"
+}
+```
+
+This will deploy a Calico node controller to every instance of the cluster
+using a Kubernetes DaemonSet. After a successful deployment you should be able
+to see these Pods running in your cluster:
+
+```
+kubectl get pods --namespace kube-system -l k8s-app=calico-node -o wide
+NAME                READY     STATUS    RESTARTS   AGE       IP             NODE
+calico-node-034zh   2/2       Running   0          2h        10.240.255.5   k8s-master-30179930-0
+calico-node-qmr7n   2/2       Running   0          2h        10.240.0.4     k8s-agentpool1-30179930-1
+calico-node-z3p02   2/2       Running   0          2h        10.240.0.5     k8s-agentpool1-30179930-0
+```
+
+Per default Calico still allows all communication within the cluster. To enable a stricter policy follow this guide: [Simple Policy Demo](http://docs.projectcalico.org/v2.0/getting-started/kubernetes/tutorials/simple-policy)

```

Comment:
I think this should reference the official k8s docs as the primary reference: https://kubernetes.io/docs/user-guide/networkpolicies/
Probably also have the link to the Calico doc as a secondary reference example walkthrough.